ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml

```diff
@@ -14,6 +14,8 @@ StackView {
     // id, name, image, color, owner properties expected
     required property var communityDetails
     required property bool isOwner
+    required property bool isTokenMasterOwner
```

Comment:
just `isTokenMaster`?

Reply:
I would like to leave it as `isTokenMasterOwner` bc there are 2 concepts here that could be mixed up:
- First one is the concept of having the `token master token` deployed (that maybe we will need to add in some point a property like `isTokenMaster` or `isTokenMasterDeployed` or `tokenMasterExists`.
- Second one is the concept of owning this type of token, so for being more explicit and to not have confusions I've named it like this: `isTokenMasterOwner`.

WDYT?